app/models/manageiq/providers/azure/cloud_manager/event_catcher.rb

```diff
@@ -5,10 +5,6 @@ class ManageIQ::Providers::Azure::CloudManager::EventCatcher < ManageIQ::Provide
   # providers that do not support it.
   #
   def self.all_valid_ems_in_zone
-    super.select do |ems|
-      ems.supports?(:timeline).tap do |available|
-        _log.info(ems.unsupported_reason(:timeline) + " [#{ems.provider_region}]") unless available
-      end
-    end
+    super.select { |ems| ems.supports?(:events) }
```

Comment:
I expected the base class to have this code.

Reply:
We could certainly do that if we go through and add `supports :events` to all providers which have an EventCatcher worker, however there aren't that many providers where collection of events is conditional at the instance level.  Most have event collection or they don't so just having an EventCatcher class is all that is required.

Unlike a lot of the core API calls where we're looking for all EMS classes which support a certain feature, we start out with the EMS worker class so if an EMS doesn't support events it won't have the class at all.